packages/graphql-toolbox/src/constants.ts

```diff
@@ -28,7 +28,7 @@ export const LOCAL_STATE_CREATE_CONSTRAINT = "neo4j.graphql.createConstraint";
 export const LOCAL_STATE_CONSTRAINT = "neo4j.graphql.constraint";
 export const LOCAL_STATE_EDITOR_THEME = "neo4j.graphql.editorTheme";
 export const LOCAL_STATE_ENABLE_REGEX = "neo4j.graphql.enable.regex";
-export const LOCAL_STATE_FAVOURITES = "neo4j.graphql.favourites";
+export const LOCAL_STATE_FAVORITES = "neo4j.graphql.favourites";
```

Comment:
Keeping the spelling for `neo4j.graphql.favourites` to not break backward compatibility.

Reply:
After a discussion with @mjfwebb, we concluded that it's best to update it now that it is still in beta